spec/features/allocations_summary_feature_spec.rb

```diff
@@ -13,7 +13,7 @@
       expect(page).to have_css('.pagination ul.links li', count: 7)
     end
 
-    it 'renders allocation offenders at index', :raven_intercept_exception, vcr: { cassette_name: :allocated_offenders } do
+    it 'renders allocation offenders at index', :raven_intercept_exception, vcr: { cassette_name: :allocated_offenders_feature } do
```

Comment:
All VCR names LGTM. However, just wondering about this file and whether they could be named slightly differently as when we add more feature specs we're could end up needing to refer to allocated/awaiting/missing info tabs and there could be some similarly named cassettes used?

Reply:
@KPobeeNorris I think rose said that this view will be reduced to one call to the controller again so I imagine we will only need one cassette in the near future. Happy to rename now though it it would make things clearer?

Reply:
Ah - no worries if it's going to be changed!